Automattic/jetpack
# Site Logo: issues when updating logo in the customizer

When site logo is replaced using the Customizer, the logo image in the site preview does not change.

Jetpack replaces the `src` attribute of the image with new value, but `srcset` remains the same, so the browser continues to display the older image.

This commit makes Jetpack remove the `srcset` attribute from the site preview logo when new logo gets chosen, so the updated `src` attribute takes effect immediately.

Fixes #18950

Props to @JessBoctor for helping with the issue.

#### Changes proposed in this Pull Request:
* Remove the `srcset` attribute from the log in Customizer's site preview.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Connect Jetpack.
2. Install the [Resonar theme](https://wordpress.org/themes/resonar/) and activate it.
3. Go to "Appearance -> Customizer" and choose "Site Identity". Upload a logo if you didn't have one, click "publish".
4. Choose another logo, make sure that the new logo appears in both the sidebar and the site preview.
5. Click "Publish", confirm that the site logo has been updated.

#### Proposed changelog entry for your changes:
* Site Logo: fixing issues when updating logo in the Customizer.
